frontend/src/components/browse-sightings-page/MapPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sighting } from "../../clients/apiClient";
+// import { SlippyMap, Marker, Label, InfoBox } from "react-slippy-map";
+import GoogleMapReact from "google-map-react";
+import { Marker } from "./marker";
+import React from "react";
+import { map } from "leaflet";
+import "./mapPage.scss";
```

Comment:
Uppercase filename please (this will play absolute havoc with the windows filesystem since it doesn't distinguish casing, so if you can't get git to play ball with this rename, come get me)
```suggestion
import "./MapPage.scss";
```